packages/contracts-periphery/contracts/universal/DrippieRedd.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import { RetroReceiver } from "./RetroReceiver.sol";
+import { console } from "hardhat/console.sol";
+
+/**
+ * @title DrippieRedd
+ * @notice DrippieRedd goes brrr.
+ */
+contract DrippieRedd is RetroReceiver {
+    /**
+     * Enum representing different status options for a given drip.
+     */
+    enum DripStatus {
+        NONE,
+        ACTIVE,
+        PAUSED
+    }
+
+    /**
+     * Represents the configuration for a given drip.
+     */
+    struct DripConfig {
+        address payable recipient;
+        bytes data;
+        bytes checkscript;
```

Comment:
Let's document usage instructions for the script here.

Reply:
Now heavily documented in comments